### PR TITLE
separate libsanitizer

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -415,6 +410,51 @@ outputs:
       home: https://gcc.gnu.org/onlinedocs/gccint/Libgcc.html
       license: GPL-3.0-only WITH GCC-exception-3.1
 
+  - name: libgcc-ng
+    target: {{ cross_target_platform }}
+    script: install-libgcc-no-gomp.sh
+    requirements:
+      build:
+        - binutils_impl_{{ target_platform }}  {{ binutils_version }}
+         binutils_impl_{{ cross_target_platform }}  {{ binutils_version }}
```
Comment (beckermr) [inline recipe/meta.yaml:419]: syntax error here

Comment (isuruf) [reply recipe/meta.yaml:419]:
```suggestion
        - binutils_impl_{{ cross_target_platform }}  {{ binutils_version }}
```

**Comment by beckermr** [comment]
I don't see anything obviously wrong. Do we need build number or other constraints so we don't mix builds that are split vs non-split.

**Comment by isuruf** [comment]
I'll mark the older builds of 9.4.0, 11.1.0, 10.3.0 as broken and add a repodata patch for older gcc builds.

**Pull request merged**